components/Filter.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Button } from "@mui/material";
```

Comment:
unfortunately we don't allow the usage of material UI. The same goes for tailwind. Please only use styled components.